prisma/schema.prisma

```diff
@@ -671,3 +675,17 @@ model OrganizationDomain {
   id     String @id @default(uuid())
   domain String
 }
+
+model SupplementalPosition {
+  id String @id @default(cuid())
+
+  organizationUnit   OrganizationUnit @relation(fields: [organizationUnitId], references: [id])
+  organizationUnitId String
+
+  user       User?   @relation(fields: [userId], references: [id])
+  userId     String?
```

Comment:
userId should not be optional

Reply:
What if we need supplemental positions for user request?

Reply:
Or we can store in user request it in Json and create SupplementalPosition when creating user?